src/routes.js

```diff
@@ -35,15 +34,9 @@ export const AppRouter = () => {
         <Route exact path="/transactions" name="Transactions" element={<Transactions/>}/>
         <Route exact path="/transaction-new" name="New Transactions" element={<TransactionNew/>}/>
         <Route exact path="/access-keys" name="Access Keys" element={<AccessKeys/>}/>
-        {(!!xPub || !!accessKeyString)
-          ?
-          <Route exact path="/" name="xPub" element={<XPub/>}/>
-          :
-          <>
-            <Route exact path="/" name="Admin dashboard" element={<Dashboard/>}/>
-          </>
+        {(!!xPub || !!accessKeyString) &&
```

Comment:
What is the 'index' view if user sign in with xPriv?

Reply:
First from the navigation so adding new xpub `/admin/register-xpub`